pkg/kubelet/cm/dra/plugin/plugins_store.go

```diff
@@ -37,43 +40,57 @@ func (s *pluginsStore) get(pluginName string) *Plugin {
 	s.RLock()
 	defer s.RUnlock()
 
-	return s.store[pluginName]
+	instances := s.store[pluginName]
+	if len(instances) == 0 {
+		return nil
+	}
+	// Heuristic: pick the most recent one. It's most likely
+	// the newest, except when kubelet got restarted and registered
+	// all running plugins in random order.
```

Comment:
Do we want to remember this info on disk to prevent random load order?

Reply:
I think this is so unlikely (kubelet being restarted in the middle of a driver update) and not harmful when it picks the older (we explicitly state that both instances must be usable interchangeably) that it doesn't justify making the code more complex.

TLDR: the heuristic should be sufficient.